compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -417,8 +417,7 @@ extern "C" LLVMAttributeRef LLVMRustCreateMemoryEffectsAttr(LLVMContextRef C,
       report_fatal_error("bad MemoryEffects.");
   }
 }
-
-// Enable a fast-math flag
+// Enable all fast-math flags
```

Comment:
It would be nice to expand this comment, and the one on `LLVMRustSetAlgebraicMath`, to give just a little bit more detail and a comparison of the two and when/why you might choose one over the other.